wrapanapi/base.py

```diff
@@ -373,6 +373,23 @@ def usage_and_quota(self):
             'Provider {} does not implement usage_and_quota'.format(type(self).__name__))
 
 
+class WrapanapiAPIBaseNetwork(WrapanapiAPIBase):
```

Comment:
**REQUIRED:** It seems weird that we copy the stats method here, perhaps we should move it to the Base instead?

Reply:
You are right. I ended up moving the `def stats` method to the `WrapanapiAPIBase` class as well as `def disconnect`. Then it turned out that we don't really need the WrapanapiAPIBaseNetwork class at all, since `NuageSystem` can extend the Base directly.